docs/instrumenting-rails.md

```diff
@@ -1,6 +1,44 @@
 # Instrumenting Rails
 
-TODO
```

Comment:
@tsloughter-splunk Any Rails application can be instrumented? Any conflict if other libraries are instrumented? Just asking because the behavior for Django apps in Python required going through lots of hoops and exceptions.

Reply:
I wouldn't expect anything like that. Do you have any examples for the common difficulties in Django?

Reply:
Lots of small caveats, but perhaps not applicable here: https://docs.splunk.com/Observability/gdi/get-data-in/application/python/instrumentation/instrument-python-frameworks.html

Reply:
I see some similarities to ruby setups from 10 years ago, not so much the modern ones, so I'd say let's just release this now, and if customers come with questions, we'll update the docs based on questions.